### DISCUSSION TITLE
想问一下顶菜单栏，点击分类目录不生效是什么原因？

### DISCUSSION THREAD

**Discussion by ghost** [solstice23/argon-theme#378] [General]
想问一下大家顶菜单栏，点击分类目录不生效是什么原因？但是右键新标签打开是有效的

domain.com/catagory/一级分类

想顶栏直接进入一级分类，但是点击不生效？

求解。

**Comment by solstice23** [comment]
feature，有二级目录的顶栏菜单没有一级分类的点击事件
